.vscode/cSpell.json

```diff
@@ -1,420 +1,436 @@
-        "reinstatable",
-        "remotable",
-        "remoting",
-        "renderable",
-        "Renormalize",
-        "renormalized",
-        "reproject",
-        "reprojected",
-        "reprojection",
-        "Resizer",
-        "Resymbolization",
-        "resymbolize",
-        "resymbolizing",
-        "retriangulate",
-        "roadmap",
-        "rpcinterface",
-        "rpcinterfaces",
-        "ruleset",
-        "rulesets",
-        "scrollbar",
-        "selectable",
-        "semibold",
-        "serializable",
-        "shadowable",
-        "skybox",
-        "snappable",
-        "Strokable",
-        "struct",
-        "structs",
-        "subclassable",
-        "subclassing",
-        "subcurve",
-        "subgraphic",
-        "submenu",
-        "subtractive",
-        "superclass",
-        "superclasses",
-        "superset",
-        "svgdotjs",
-        "svgjs",
-        "sweepable",
-        "symb",
-        "symbologies",
-        "symbology",
-        "symbology's",
-        "syncui",
-        "system",
-        "tabindex",
-        "tbgr",
-        "Tesselated",
-        "testid",
-        "texel",
-        "texels",
-        "textbox",
-        "themecolors",
-        "theming",
-        "tileset",
-        "tilesets",
-        "to",
-        "toleranced",
-        "tolerancing",
-        "toolassistance",
-        "Topo",
-        "tranposed",
-        "transpiler",
-        "transpiling",
-        "tridiagonal",
-        "tsnode",
-        "tween",
-        "tweening",
-        "Tweens",
-        "Txns",
-        "uicore",
-        "uifw",
-        "Uint",
-        "uiprovider",
-        "uisettings",
-        "Ulas",
-        "unbordered",
-        "uncheck",
-        "unchecking",
-        "unclipped",
-        "Uncompiled",
-        "underbar",
-        "Undisplayable",
-        "undoable",
-        "undock",
-        "Unexpanded",
-        "unflipped",
-        "ungap",
-        "ungroup",
-        "unhilite",
-        "unhilited",
-        "unhilites",
-        "Uninstanced",
-        "unitless",
-        "unmanaged",
-        "unmount",
-        "unmounting",
-        "unmounts",
-        "unnormalized",
-        "unoriented",
-        "unquantize",
-        "unsuspended",
-        "Untextured",
-        "untransposed",
-        "unweight",
-        "UORs",
-        "Uploader",
-        "upvote",
-        "userlabel",
-        "uv",
-        "vect",
-        "viewbox",
-        "viewflags",
-        "viewlet",
-        "Viewport's",
-        "viewports",
-        "viewstate",
-        "webgl",
-        "weightpicker",
-        "Wireframe",
-        "wiremesh",
-        "WMTS",
-        "Wrappable",
-        "xyxy",
-        "XYZAs",
-        "XYZH",
-        "xyzuv",
-        "XYZUVW",
-        "XYZW",
-        "XYZXYZ",
-        "xyzxyzxyz",
-        "zbuffer"
-    ],
-    "flagWords": [
-        "hte",
-        "minium"
-    ]
+  "version": "0.2",
+  "language": "en",
+  "enableFiletypes": [
```

Comment:
just moved this out of the settings.json into here to keep related configs together